examples/registration_form/lib/address/address_component.html

```diff
@@ -0,0 +1,42 @@
+<div ngControlGroup="address">
+    <material-input
+            type="text"
+            floatingLabel
+            label="Address 1"
+            ngControl="address1"
+            [(ngModel)]="address1"
+            required
+            #address1Control="ngForm">
```

Comment:
Maybe comment this line for users following along? This is a pattern I really like to see, but I think the casual observer would probably not understand what is going on here.

Reply:
Done